ScoutSuite/frontend/src/pages/Resources/style.scss

```diff
@@ -0,0 +1,20 @@
+@import 'src/variables.scss';
```

Comment:
i copied the style from the Findings for the Resources but we should find a way so they can share it.

Reply:
Maybe we could have a styles directory and import the shared files from there.